mainapp/tasks/__init__.py

```diff
@@ -234,11 +234,12 @@ def check_notify_metric_changed_task(metric_id: UUID) -> None:
         ):
             # Send slack message
             logger.info(f"Sending slack notification for metric_id={metric_id}")
+            link = f"https://polynomial.so/metrics/{metric_id}/embed"
```

Comment:
Hi @madsnedergaard,
Thanks for the contribution!
I added a new `metric-details` route [here](https://github.com/corradio/polynomial/commit/ba1997306ad6f18bbc8fab20c18e2d34cf6bac40#diff-742b0a5d9e820d2640675ab798fc9eec0ffdb9079cf908e84ee519c105614d9eR625), which is the same as `/embed` but with proper access rights and with header/footer for the UI.

I also suggest constructing the url from the named routes:
```suggestion
            link = f"{BASE_URL}{reverse('metric-details', args=[metric_id])}"
```

using:
```python
from config.settings import CSRF_TRUSTED_ORIGINS

BASE_URL = CSRF_TRUSTED_ORIGINS[0]
```

The `BASE_URL` is a bit hacky but at least it's consistent with [this](https://github.com/corradio/polynomial/blob/master/mainapp/tasks/error_handling.py#L45).

Reply:
Hi @madsnedergaard , let me know if you need anything from me!

Reply:
Sorry it took a while, should be fixed now - I used the `reverse()` method used elsewhere, although I'm not 100% certain how that actually works ¯\\\_(ツ)\_/¯

Reply:
FYI it gets the URL by key, as defined in urls.py here:
https://github.com/corradio/polynomial/blob/master/mainapp/urls.py#L34